pkg/elle/rw_register/op.go

```diff
@@ -0,0 +1,77 @@
+package rwregister
+
+import (
+	"regexp"
+	"strconv"
+
+	"github.com/pingcap/tipocket/pkg/elle/core"
+)
+
+var (
+	opPattern = regexp.MustCompile(`([rw])([a-zA-Z])([0-9_]+)(.*)`)
```

Comment:
so 1 means "w/r"m, 2 means "key", 3means "keynumber", can you give some string samples here?

Reply:
`rx1` => `[:r :x 1]`
`rx_` => `[:r :x nil]`
`wx1` => `[:w :x 1]`